.gitignore

```diff
@@ -105,6 +105,7 @@ auto/
 
 #gregorio specific
 doc/_minted-*
+doc/*.pyg
```

Comment:
This is now redundant with a modification that was made as part of 4.2 (where all `*.pyg` files are ignored, not just the ones in `doc`).